src/webui/www/public/preferences_content.html

```diff
@@ -81,7 +81,9 @@
     <li>QBT_TR(%Z: Torrent size (bytes))QBT_TR</li>
     <li>QBT_TR(%T: Current tracker)QBT_TR</li>
     <li>QBT_TR(%I: Info hash)QBT_TR</li>
-    </ul></i>
+    </ul>
+    QBT_TR(Tip: Encapsulate parameter with quotation marks to avoid text being cut off at whitespace (e.g., "%N"))QBT_TR
```

Comment:
Are you sure you don't have to escape `(e.g., "%N")` like it's done in the C++ code(`(e.g., \"%N\")`)?